docs/topic-guides/blendable-web-apps/sessions.md

```diff
@@ -100,7 +100,7 @@ Handle.GET("/usesession/{?}", (Session session, Request request) =>
 });
 ```
 
-* Session Cookie: Add a header on outgoing response by setting property `UseSessionHeader` to `true` and optionally specify name of header with `SessionHeaderName` \(default `X-Location`\).
+* Unique session URL: Add an HTTP header on outgoing response by setting property `UseSessionHeader` to true and optionally specify name of header with `SessionHeaderName` (default `X-Location`).
```

Comment:
Thanks @alshakero for spotting this out, you are right that Session cookie is not supported in 2.4.

But there is no point in keeping this bullet point because it repeats information from the first bullet point (`Location` + `X-Referer` or `Referer`).

Reply:
Has anyone tested if `SessionHeaderName` is actually working? Because I wonder how gateway knows about it.

Reply:
It was added in this commit: https://github.com/Starcounter/level1/commit/5a5d25ac98e5c0848f2eae9910949b03a3c2cb18